### PR TITLE
2986_As_a_3rd-Party_integrator

### PR DESCRIPTION
**Pull request by lilitkarapetyan**
## 🔨Work carried out:

- [x] Tests pass

## Confirmations

- [ ] I have chosen reviewers for my PR.
- [ ] I have assigned myself to this PR.
- [ ] I have chosen an appropriate label for the PR.
- [ ] I have completed the mandatory sections of this document.
- [ ] I have deleted any unused sections.
- [ ] I confirm that I have checked for required README updates and acted accordingly.

### CODE REVIEW

**Comment by lilitkarapetyan** [comment]
Fixes #2986

**Comment by IanMayo** [comment]
lilit, it isn't easy to review this. Are we hosting the pages in heroku deployment for this PR? Can I have a link?

**Comment by lilitkarapetyan** [comment]
> lilit, it isn't easy to review this. Are we hosting the pages in heroku deployment for this PR? Can I have a link?

Yes I will send the heroku url now

**Comment by lilitkarapetyan** [comment]
> lilit, it isn't easy to review this. Are we hosting the pages in heroku deployment for this PR? Can I have a link?

Yes I will send the heroku url now - https://serge-2986-as-a-3rd-par-nypcs9.herokuapp.com/api-docs/

**Comment by IanMayo** [comment]
This is great, thanks Lilit.

1. Do we have to run the server in a special mode to server the documents, or is there a way for the Swagger API to always be visible?  (I don't think it really consumes much CPU)
2. Does `couchdb.js need to be documented?
3. Note: the `storybook` database should be deleted:
image

**Comment by lilitkarapetyan** [comment]
> This is great, thanks Lilit.
> 
>     1. Do we have to run the server in a special mode to server the documents, or is there a way for the Swagger API to always be visible?  (I don't think it really consumes much CPU)
> 
>     2. Does `couchdb.js need to be documented?
> 
>     3. Note: the `storybook` database should be deleted:
>        image

Hi @IanMayo , Ir also works for couchDB

**Comment by IanMayo** [comment]
> It also works for couchDB

Aah, is that because both backends use the same REST API structure, right?

File: server/package.json [hunk 1]
```diff
@@ -40,6 +42,7 @@
   "scripts": {
     "dev": "nodemon main.js",
     "start": "node main.js",
+    "serve-openapi": "node main.js --openapi",
```
Comment (IanMayo) [inline server/package.json:45]: @lilitkarapetyan - is the `serve-openapi` command there _just-in-case_?

So, the API docs are always there, but this command is to just serve the API docs.  Is that right?  If it is, we should add a some comment to the read-me.

**Comment by IanMayo** [comment]
Could we have a link in Serge to open the OpenAPI docs? 

I suggest it's another link in the admin pages.  I suggest it's a new button in the left-hand menu of the `Admin` page.  It would go beneath `WELCOME SCREEN`, and be titled `REST API`.

**Comment by lilitkarapetyan** [comment]
> Could we have a link in Serge to open the OpenAPI docs?
> 
> I suggest it's another link in the admin pages. I suggest it's a new button in the left-hand menu of the `Admin` page. It would go beneath `WELCOME SCREEN`, and be titled `REST API`.

@IanMayo , Okay I will add now. 
in this section ?
Screenshot (302)

**Review by IanMayo — APPROVED** [review]
Tested and working

**Pull request merged**